### ISSUE TITLE
Couldnt locate 4th bloodstone, round instantly won on ACT IV, no Narsie

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#20917]
(WEB REPORT BY: chazach REMOTE: 206.221.180.138:7777)
# Revision
6ee59bec7b4132fad966658c4fa4b6a0f0c80da5

# Description
No 4th stone to be found, as soon as the centcomm announcement came, narsie was summoned. Also he wasnt since he didnt appear

**Comment by ShiftyRail** [comment]
Related runtime : 
```dm
[16:52:42] Runtime in bloodcult_buildings.dm,1301: Cannot read null.z
  proc name: initialize holomap (/datum/station_holomap/cult/initialize_holomap)
  src: /datum/station_holomap/cult (/datum/station_holomap/cult)
  call stack:
  /datum/station_holomap/cult (/datum/station_holomap/cult): initialize holomap(null, null, null)
  Blood Cult: ACT IV (/datum/objective/bloodcult_tearinreality): PostAppend()
  /datum/objective_holder (/datum/objective_holder): AddObjective(Blood Cult: ACT IV (/datum/objective/bloodcult_tearinreality), null, Cult of Nar-Sie (/datum/faction/bloodcult))
  Cult of Nar-Sie (/datum/faction/bloodcult): AppendObjective(Blood Cult: ACT IV (/datum/objective/bloodcult_tearinreality), 0)
  Cult of Nar-Sie (/datum/faction/bloodcult): progress(4, null)
  Cult of Nar-Sie (/datum/faction/bloodcult): add bloody floor(space (175,190,1) (/turf/space))
  the drips of blood (/obj/effect/decal/cleanable/blood/drip): New(space (175,190,1) (/turf/space))
  getFromPool(/obj/effect/decal/cleanable/bl... (/obj/effect/decal/cleanable/blood/drip), space (175,190,1) (/turf/space))
  blood splatter(Percival Fuchs (/mob/living/carbon/human), Blood (/datum/reagent/blood), null)
  Percival Fuchs (/mob/living/carbon/human): drip(3.99375)
  Percival Fuchs (/mob/living/carbon/human): handle blood()
  Percival Fuchs (/mob/living/carbon/human): handle regular status updates()
  Percival Fuchs (/mob/living/carbon/human): Life()
  Mob (/datum/subsystem/mob): fire(1)
  Mob (/datum/subsystem/mob): ignite(1)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing()
```
Invisible Nar-Sie runtime (maybe ?)
```dm
[16:52:43] Runtime in narsie.dm,173: Cannot read null.icon_state
  proc name: narsiefloor (/obj/machinery/singularity/narsie/proc/narsiefloor)
  src: Nar-Sie (/obj/machinery/singularity/narsie/large)
  src.loc: null
  call stack:
  Nar-Sie (/obj/machinery/singularity/narsie/large): narsiefloor(null)
  Nar-Sie (/obj/machinery/singularity/narsie/large): move(0)
```

Essentially, it tries to skip to Act IV => Create an holominimap => Can't, so it runtimes => The objective is never added => It counts as a cult victory
Nar-Sie arrived the second after, so it can't be a coincidence

Looks like this runtime can also trigger in other circumstances : 
```dm
[16:38:51] Runtime in bloodcult_buildings.dm,1301: Cannot read null.z
  proc name: initialize holomap (/datum/station_holomap/cult/initialize_holomap)
  usr: Ray Bashy Jr. (arbie) (/mob/living/carbon/human)
  usr.loc: The engraved floor (210, 312, 1) (/turf/simulated/floor/engine/cult)
  src: /datum/station_holomap/cult (/datum/station_holomap/cult)
  call stack:
  /datum/station_holomap/cult (/datum/station_holomap/cult): initialize holomap(null, null, null)
  the blood stone (/obj/structure/cult/bloodstone): Destroy()
  qdel(the blood stone (/obj/structure/cult/bloodstone), 0, 0)
  the blood stone (/obj/structure/cult/bloodstone): takeDamage(30)
  the blood stone (/obj/structure/cult/bloodstone): bullet act(the laser (/obj/item/projectile/beam), "chest")
  the laser (/obj/item/projectile/beam): to bump(the blood stone (/obj/structure/cult/bloodstone), 1)
  the engraved floor (211,314,1) (/turf/simulated/floor/engine/cult): Enter(the laser (/obj/item/projectile/beam), the engraved floor (211,313,1) (/turf/simulated/floor/engine/cult))
  the laser (/obj/item/projectile/beam): Move(the engraved floor (211,314,1) (/turf/simulated/floor/engine/cult), 0, 0, 0, 0)
  the laser (/obj/item/projectile/beam): bresenham step(2, 1, 1, 4, the engraved floor (211,313,1) (/turf/simulated/floor/engine/cult), null, "\[0x2027896]")
  the laser (/obj/item/projectile/beam): process()
```

**Comment by DeityLink** [comment]
Fixed by #20919
@ShiftyRail
